File: data simulator/models/follow.py
class Follow:
    def __init__(self):
        self.seller_id = None
        self.user_id = None
        
    def __str__(self):
        return f"""
        {
            "seller_id" : "{self.seller_id}", 
            "user_id" : "{self.user_id}"
        }
    """

File: server/src/main/kotlin/server/server/models/Follow.java
package server.server.models;

import jakarta.persistence.*;
import lombok.*;
import server.server.models.compositeKeys.FollowKey;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Follow {

    @EmbeddedId
    private FollowKey followKey;

    @ManyToOne
    @JoinColumn(name="user_id",insertable = false, updatable = false)
    private User user;

    @ManyToOne
    @JoinColumn(name="seller_id",insertable = false, updatable = false)
    private Seller seller;

}
